app/javascript/hyacinth_v1/components/shared/forms/buttons/SubmitButton.js

```diff
@@ -2,17 +2,15 @@ import React from 'react';
 import { Button } from 'react-bootstrap';
 import { withRouter } from 'react-router-dom';
 
-class SubmitButton extends React.PureComponent {
-  render() {
-    // Note: Extracting staticContext here won't be necessary when we
-    // switch to functional components + hooks
-    const { formType, staticContext, ...rest } = this.props;
-    return (
-      <Button variant="primary" type="submit" {...rest}>
-        {formType === 'new' ? 'Create' : 'Update'}
-      </Button>
-    );
-  }
-}
+const SubmitButton = (props) => {
+  // Note: Extracting staticContext here won't be necessary when we
+  // switch to functional components + hooks
+  const { formType, staticContext, ...rest } = props;
+  return (
+    <Button variant="primary" type="submit" {...rest}>
+      {formType === 'new' ? 'Create' : 'Update'}
+    </Button>
+  );
+};
 
 export default withRouter(SubmitButton);
```

Comment:
Same question here about whether withRouter should still be used now that this is a functional component.  On this branch, this `withRouter ` occurrence and the other one I mentioned above are the only ones that are left in the app.  I think okay to remove?

Reply:
Removed.